test/abi/Inputs/macOS/arm64/stdlib/baseline

```diff
@@ -13625,6 +13625,7 @@ _swift_getExtendedExistentialTypeMetadata
 _swift_getExtendedExistentialTypeMetadata_unique
 _swift_getExtendedExistentialTypeShape
 _swift_getExtendedFunctionTypeMetadata
+_swift_getFixedArrayTypeMetadata
```

Comment:
For these tests, you should add an `Added: _swift_getFixedArrayTypeMetadata` entry to both `abi/macOS/arm64/stdlib.swift` and `abi/macOS/x86_64/stdlib.swift`